build.sbt

```diff
@@ -10,12 +10,19 @@ organization := "ohnosequences"
 
 bucketSuffix := "era7.com"
 
+resolvers ++= Seq(
+  "Era7 maven releases" at "http://releases.era7.com.s3.amazonaws.com"
+)
 
-// plugins which will be inherrited by anybody who uses this plugin:
-addSbtPlugin("ohnosequences" % "sbt-s3-resolver" % "0.7.0")
+addSbtPlugin("ohnosequences" % "sbt-s3-resolver" % "0.10.1")
 
-addSbtPlugin("ohnosequences" % "nice-sbt-settings" % "0.3.1")
+addSbtPlugin("ohnosequences" % "nice-sbt-settings" % "0.4.0-RC2")
```

Comment:
this is a strange version.. there is `0.6.0-SNAPSHOT` at the moment (I should make a release)